indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), description=_("Location"))
```

Comment:
I think manually generating the HTML is cleaner. And since this widget has already so much JS, another line will not harm.